### ISSUE TITLE
Using overrides to disable some rules fails

### ISSUE THREAD

**Issue by antusus** [stoplightio/spectral#2469]
**Chore summary**
I'm using spectral as a JS dependency in version `5.9.2`. I'm validating output json file and some paths have lots of errors that I must postpone addressing. I wanted to use overrides to disable rules but it is not working and I have no idea what I'm doing wrong.

In my root folder I have

## `.spectral.yml`

```yaml
extends: [[spectral:oas, all]]

overrides:
  - files:
      - "**#/components/schemas/Activity/properties/tags/example"
    rules:
      oas3-valid-schema-example : "info"
```

## `openapi.json`

I'm including only very small portion of my file
```json
{
  "openapi": "3.0.2",
  "info": {
    "title": "Test",
    "version": "1.0.0"
  },
  "paths": {
  },
  "components": {
    "schemas": {
      "Activity": {
        "title": "Activity",
        "type": "object",
        "properties": {
          "tags": {
            "type": "array",
            "example": "1"
          }
        }
      }
    }
  }
}

```

The example is not a proper array so the validation fails as expected with:
```
 17:24    error  oas3-valid-schema-example      `example` property type should be array                                  components.schemas.Activity.properties.tags.example
```

But I specified that this should be reported as `info` and should not fail my test.

I'm using command
```
spectral lint openapi.json -F error --fail-on-unmatched-globs -v
```

Can you tell me what I'm doing wrong? 
Can I disable the `oas3-valid-schema-example` rule?
I was trying with path: `"openapi.json#/components/schemas/Activity/properties/tags/example"` and it also fails.

**Additional context**
If needed I've created sample project here: https://github.com/antusus/spectral-test and you should be able to use any node version.

**Comment by P0lip** [comment]
Hey!
You should use Spectral 6.x. 5.9.2 does not support overrides.
With 6.x it works just fine.
<img width="1153" alt="image" src="https://github.com/stoplightio/spectral/assets/9273484/855d14b5-83cc-4ba4-b816-376e0a16970d">

If you chose npm/yarn installation, make sure to remove `@stoplight/spectral` and install `@stoplight/spectral-cli` instead.

**Comment by antusus** [comment]
Hi,
@P0lip any suggestion what I can use to ignore those errors in version 5 of spectral? 
I cannot easily upgrade to v6... looks like I have some dependency clashes.

**Comment by P0lip** [comment]
Spectral v5 only lets you disable the rule entirely via [exceptions](https://github.com/stoplightio/spectral/blob/014806269c4b533264c9529bebfc1c76312782e3/docs/getting-started/rulesets.md#selectively-silencing-some-results). You'll need to upgrade to v6 if you want to leverage overrides.

**Comment by antusus** [comment]
Thanks for all the answers.
